docker/scripts/osie.sh

```diff
@@ -542,8 +542,14 @@ if [[ $pwhash == "preinstall" ]]; then
 	exit 0
 fi
 
-echo -e "${GREEN}#### Setting root password${NC}"
-set_root_pw "$pwhash" $target/etc/shadow
+echo -e "${GREEN}#### Setting password${NC}"
+if [[ ${OS} =~ vmware_nsx_3_0_0 ]]; then
+	pwuser="admin"
+	set_pw "$pwuser" "$pwhash" $target/etc/shadow
+else
+	pwuser="root"
+	set_pw "$pwuser" "$pwhash" $target/etc/shadow
+fi
```

Comment:
since only pwuser changes, lets only mess with that. Also, it should be ok to be a bit more generic and support all vmware_nsx values right? So:
```sh
pwuser=root
[[ ${OS} =~ ^vmware_nsx ]] && pwuser=admin

set_pw "$pwuser" "$pwhash" $target/etc/shadow 
```

Reply:
Yep it makes sense to only mess with that. Nope, other/older versions of NSX appear to use root for default.